src/Notecard.cpp

```diff
@@ -245,6 +253,15 @@ void Notecard::begin(NoteSerial * noteSerial_)
     if (noteSerial) {
         NoteSetFnSerial(noteSerialReset, noteSerialTransmit,
                         noteSerialAvailable, noteSerialReceive);
+
+        // Set the default debug serial throttling
+        J *req = NoteNewRequest("card.aux.serial");
+        if (req != NULL)
+        {
+            JAddIntToObject(req, "max", SERIAL_RX_BUFFER_SIZE - 1);
```

Comment:
What's the -1 for?

Reply:
We observed that when the buffer is filled entirely, then it chokes on the last byte.